packages/hyperion-autologging/src/ALInteractableDOMElement.ts

```diff
@@ -524,14 +534,23 @@ export function getElementTextEvent(
   surface: string | null,
   // Event name to utilize when attempting to resolve text from a parent interactable
   // Some interactable elements may have no text to extract, in those cases we want to move up the tree to attempt from parent interactable.
-  tryInteractableParentEventName?: UIEventConfig['eventName'] | null
+  tryInteractableParentEventName?: UIEventConfig['eventName'] | null,
+  skipCache?: boolean
```

Comment:
nit: Maybe rename to `useCacheText`, I initially got confused why there are two variables for using the text cache (also `cacheText`)